installer.sh

```diff
@@ -270,9 +293,10 @@ function deploy_frontend {
   EXISTS=$(check_app_exists ${APP_NAME})
   if [ $EXISTS -eq 1 ]
   then
-    cf d ${APP_NAME} -f
+    RETURN=($(cf d ${APP_NAME} -f))
+	check_return
   fi 
-  git clone ${GITHUB_SPACE}/iotanalytics-dashboard.git &&
+  git clone ${GITHUB_SPACE}/iotanalytics-dashboard.git
```

Comment:
you removed "&&" which means that even if git clone will fail, it will try do next step, I think we should place it again there